fieldservice_fsm_person_salary_extended/README.rst

```diff
@@ -0,0 +1,57 @@
+=======================================
+FieldService FSM Person Salary Extended
+=======================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-solvosci%2Fslv--field--service-lightgray.png?logo=github
+    :target: https://github.com/solvosci/slv-field-service/tree/15.0/fieldservice_fsm_person_salary_extended
+    :alt: solvosci/slv-field-service
+
+|badge1| |badge2| |badge3| 
+
+Adds salary fields in fsm.order.salary model.
```

Comment:
Change the summary to something simpler, like Adds new model fsm.order.salary that relates to orders and persons
And regenerate